tests/MongoDB.EntityFrameworkCore.FunctionalTests/Entities/Guides/GuidesDbContext.cs

```diff
@@ -42,10 +43,19 @@ public static GuidesDbContext Create(
 
     protected override void OnModelCreating(ModelBuilder modelBuilder)
     {
-        base.OnModelCreating(modelBuilder);
-        modelBuilder.Entity<Moon>()
-            .ToCollection("moons")
-            .HasKey(nameof(Moon.planetId), nameof(Moon.label));
-        modelBuilder.Entity<Planet>().ToCollection("planets");
+        var planetId = ObjectId.GenerateNewId();
```

Comment:
Should probably include some test coverage to ensure owned entities are correctly handled.